include/glow/Graph/Node.h

```diff
@@ -145,6 +145,9 @@ class Node : public Named,
 
   ~Node() { releaseUsers(); }
 
+  /// \returns an array ref of the types of the node.
```

Comment:
based on the comment it's not clear whether it's input/output or both.

Reply:
Done